demo/api/src/content-generation.module.ts

```diff
@@ -0,0 +1,84 @@
+import { ContentGenerationRequest, Options } from "@comet/cms-api/lib/content-generation/content-generation.types";
+
+export const ContentGenerationModels = ({ apiKey, apiUrl }: { apiKey: string; apiUrl: string }) => {
+    const createMessages = (options: ContentGenerationRequest, imageDetail: "low" | "high") => {
+        const messages = [];
+        options.instructions && messages.push({ role: "system", content: options.instructions });
+        options.context && messages.push({ role: "user", content: options.context });
```

Comment:
nit: I personally don't like the `&&` notation. I prefer ifs as they are more specific and easier to read